mods/mods_here.txt

```diff
@@ -1,4 +0,0 @@
-You can install Minetest mods by copying (and extracting) them into this folder.
-To enable them, go to the configure world window in the main menu or write
-  load_mod_<modname> = true
-in world.mt in the world directory.
```

Comment:
If you remove this file, the whole mods directory will disappear aswell. Shouldn't we at least leave this file existent to keep the structure?